src/components/Posts.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import "../App.css";
+
+const Posts = props => {
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    fetchPosts();
+  }, []);
+
+  const fetchPosts = async () => {
+    const response = await fetch("http://localhost:5000/posts/");
+    const body = await response.json();
+    const postsHtml = body.map(postObject => postObject.html);
+    setPosts(postsHtml);
+  };
+
+  const createMarkup = html => {
+    return { __html: html };
```

Comment:
Would be nice if we sanitize this html to avoid security issues. Maybe by this library: https://www.npmjs.com/package/sanitize-html

Reply:
function removed in future commits
will not use dangerouslySetInnerHTML, so no need to sanitize